contracts/bridge/x/v4/README.md

```diff
@@ -0,0 +1,8 @@
+# Immutable X Contracts
```

Comment:
Could you update the README file to have a format a bit like this one:
https://github.com/immutable/contracts/blob/5180f761415e475aee77daacc465e138893f4d58/contracts/allowlist/README.md

That is, paragraph explaining the contents of the directory, a section about audits: saying there are none is where things are for the contract, a section about where the contracts have been been deployed (similar to what you have), and then a section which briefly describes each contract in the directory.